### PR TITLE
Added sms77 as a sms provider & fix docs typo

### PR DESCRIPTION
**Pull request by matthiez**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
 Added sms77 as sms provider & fixed small typo in docs

- **What is the current behavior?** (You can also link to an open issue here)
 Missing sms77 as sms provider

- **What is the new behavior (if this is a feature change)?**
 Possibility to use sms77 as sms provider

### CODE REVIEW

File: docs/docs/community/create-provider.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ cd packages/core && yarn run build
 After the project is initialized creating a new provider is achievable with the following.
 
 ```zsh
-yarn run generate:provide
+yarn run generate:provider
```
Comment (scopsy) [inline docs/docs/community/create-provider.md:33]: Great catch! Thanks!

File: providers/sms77/src/lib/sms77.provider.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@notifire/core';
+
+import Sms77Client, {SmsJsonResponse, SmsParams} from 'sms77-client'
```
Comment (scopsy) [inline providers/sms77/src/lib/sms77.provider.ts:8]:
```suggestion
import Sms77Client, { SmsJsonResponse, SmsParams } from 'sms77-client';
```

Could you please run prettier on this file? 💅

**Review by scopsy — COMMENTED** [review]
@matthiez Looks amazing! thanks for the contribution, could you just run prettier on the provider file so we can have consistent styling across the project? 🎉

File: providers/sms77/src/lib/sms77.provider.ts (hunk 2 above)
Comment (matthiez) [reply providers/sms77/src/lib/sms77.provider.ts]: Just did that, should be fine now.
Maybe it could be helpful to add prettier to the pre-commit hook?

Comment (scopsy) [reply providers/sms77/src/lib/sms77.provider.ts]: @matthiez there should be a lint-staged pre-commit hook that does prettier. Looks like it's not working :( Could you open an issue for that?

**Comment by scopsy** [comment]
Looks awesome! Merging this to `v0.3.0`

File: providers/sms77/src/lib/sms77.provider.ts (hunk 2 above)
Comment (matthiez) [reply providers/sms77/src/lib/sms77.provider.ts]: I'm playing around a bit for now trying to fix this. I have never really used lint-staged so there is that. It seems to pick up the configuration from notifire/libs/sdk/package.json and not the actual package.json from the project.

Besides this issue, I noticed that the provider generator generated .eslintrc.js - shouldn't it be .eslintrc.json like with the existing providers?

Comment (scopsy) [reply providers/sms77/src/lib/sms77.provider.ts]: Yes, it's a mistake. @galezra already fixed that at v0.3.0 hopefully I'll merge it tomorrow or today.

**Pull request merged**